contrib/docker/README.md

```diff
@@ -9,13 +9,7 @@ use that server.
 
 ## Build
 
-Build the docker image with the `docker build` command from the root of the synapse repository.
-
-```
-docker build -t docker.io/matrixdotorg/synapse .
-```
-
-The `-t` option sets the image tag. Official images are tagged `matrixdotorg/synapse:<version>` where `<version>` is the same as the release tag in the synapse git repository.
+Build the docker image with the `docker-compose build` command.
```

Comment:
perhaps we should document that people can skip building and specify the version where we have `latest` in the docker-compose file, to pull it from docker hub.

Reply:
Yes, full ACK. But my intention is to fix this later in a separate PR. This is my first one and I wanted to get the PR process known first.